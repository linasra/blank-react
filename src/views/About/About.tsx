import { Container, Title, NavLink } from "views/viewStyles";

function About() {
  return (
    <Container>
      <Title>About</Title>
      <NavLink to="/">Home</NavLink>
    </Container>
  );
}

export default About;
