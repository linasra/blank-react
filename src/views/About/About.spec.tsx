import { render } from "@testing-library/react";
import { MemoryRouter } from "react-router-dom";
import About from "./About";

describe("About view", () => {
  it("should render the component correctly and match snapshot", () => {
    const { container } = render(
      <MemoryRouter>
        <About />
      </MemoryRouter>
    );
    expect(container).toMatchSnapshot();
  });
});
