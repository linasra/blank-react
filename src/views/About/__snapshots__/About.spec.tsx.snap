// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About view should render the component correctly and match snapshot 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  height: 100vh;
  width: 100vw;
}

.c1 {
  font-size: 2rem;
}

.c2 {
  color: #0074d9;
  -webkit-text-decoration: none;
  text-decoration: none;
  font-size: 1rem;
}

.c2:hover {
  -webkit-text-decoration: underline;
  text-decoration: underline;
}

<div>
  <div
    class="c0"
  >
    <h1
      class="c1"
    >
      About
    </h1>
    <a
      class="c2"
      href="/"
    >
      Home
    </a>
  </div>
</div>
`;
